lib/odometry-ng/OdometerDifferential.cpp

```diff
@@ -31,7 +31,7 @@ int OdometerDifferential::update()
     
     // Save polar pose delta since last call
     polar_.set_distance(delta_linear_pose);
-    polar_.set_angle(delta_angular_pose);
+    polar_.set_angle(RAD2DEG(delta_angular_pose));
```

Comment:
I think that degrees should come to and from the RPi, but inside it should be radians.

Reply:
I agree with you but I created a special issue for this https://github.com/cogip/mcu-firmware/issues/182 because this requires some additional changes that is out of this PR scope in my opinion.

Reply:
ok good to me. i resolve.